mkmapdata/runscript_mkmapdata.sh

```diff
@@ -32,7 +32,7 @@ rawpath="/p/scratch/cslts/poll1/rawdata/clm5/mkmapdata/"
 OUTPUT="$PWD"
 
 ### loop over all input files
-for file in SCRIPgrid_0.5x0.5_AVHRR_c110228.nc SCRIPgrid_0.25x0.25_MODIS_c170321.nc SCRIPgrid_3minx3min_LandScan2004_c120517.nc SCRIPgrid_3minx3min_MODISv2_c190503.nc SCRIPgrid_3minx3min_MODISwcspsea_c151020.nc SCRIPgrid_3x3_USGS_c120912.nc SCRIPgrid_5x5min_nomask_c110530.nc SCRIPgrid_5x5min_IGBP-GSDP_c110228.nc SCRIPgrid_5x5min_ISRIC-WISE_c111114.nc SCRIPgrid_5x5min_ORNL-Soil_c170630.nc SCRIPgrid_10x10min_nomask_c110228.nc SCRIPgrid_10x10min_IGBPmergeICESatGIS_c110818.nc SCRIPgrid_3minx3min_GLOBE-Gardner_c120922.nc SCRIPgrid_3minx3min_GLOBE-Gardner-mergeGIS_c120922.nc SCRIPgrid_0.9x1.25_GRDC_c130307.nc SCRIPgrid_360x720_cruncep_c120830.nc UGRID_1km-merge-10min_HYDRO1K-merge-nomask_c130402.nc
+for file in SCRIPgrid_0.5x0.5_AVHRR_c110228.nc SCRIPgrid_0.25x0.25_MODIS_c170321.nc SCRIPgrid_0.5x0.5_MODIS_c110228.nc SCRIPgrid_3minx3min_LandScan2004_c120517.nc SCRIPgrid_3minx3min_MODISv2_c190503.nc SCRIPgrid_3minx3min_MODISwcspsea_c151020.nc SCRIPgrid_3x3_USGS_c120912.nc SCRIPgrid_5x5min_nomask_c110530.nc SCRIPgrid_5x5min_IGBP-GSDP_c110228.nc SCRIPgrid_5x5min_ISRIC-WISE_c111114.nc SCRIPgrid_5x5min_ORNL-Soil_c170630.nc SCRIPgrid_10x10min_nomask_c110228.nc SCRIPgrid_10x10min_IGBPmergeICESatGIS_c110818.nc SCRIPgrid_3minx3min_GLOBE-Gardner_c120922.nc SCRIPgrid_3minx3min_GLOBE-Gardner-mergeGIS_c120922.nc SCRIPgrid_0.9x1.25_GRDC_c130307.nc SCRIPgrid_360x720_cruncep_c120830.nc UGRID_1km-merge-10min_HYDRO1K-merge-nomask_c130402.nc 
```

Comment:
I am fine with the extension, but it should already run without the extension. The MODIS data should have been already for 0.25x025 degree already (`SCRIPgrid_0.25x0.25_MODIS_c170321.nc`). Which command do you use for `mksurfdata.pl`?

Reply:
@odombro It looks to me that there are now two MODIS data processed `SCRIPgrid_0.25x0.25_MODIS_c170321.nc` and  `SCRIPgrid_0.5x0.5_MODIS_c110228.nc `. In which case do we need two MODIS data with different resolution?